operator/pkg/operator.go

```diff
@@ -646,13 +646,12 @@ func (o *Operator) SendTelemetryData(ctx *cli.Context) error {
 	hash.Write([]byte(ctx.App.Version))
 
 	// get hash
-	version := hash.Sum(nil)
+	var version [32]byte // All zeroed initially
+	copy(version[:], hash.Sum(nil))
```

Comment:
I believe signing only the version makes us susceptible to replay attacks. Can anyone with a firmer grasp of cryptography take a look? @MauroToscano @JuArce 
Note if that's true the issue was already there, it's not introduced by this PR, just preserved.

Reply:
It's not, it goes encrypted to the net, and it's for our services. We can fake being an operator and break our telemetry but we won't do that